catalog/spatial/kml/spatial-kml-transformer/src/main/java/org/codice/ddf/spatial/kml/transformer/KMLTransformerImpl.java

```diff
@@ -120,14 +94,14 @@
 
   protected BundleContext context;
 
-  private JAXBContext jaxbContext;
-
   private ClassPathTemplateLoader templateLoader;
 
   private KmlStyleMap styleMapper;
 
   private DescriptionTemplateHelper templateHelper;
 
+  private KmlMarshaller kmlMarshaller = new KmlMarshaller();
```

Comment:
Is there a reason not to instantiate this in blueprint?

Reply:
And then pass it in to the constructor?

Reply:
Or set it as a property